Xamarin.Forms.Platform.UAP/CheckBoxRenderer.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.ComponentModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	public class CheckBoxRenderer : ViewRenderer<CheckBox, FormsCheckBox>
+	{
+		static Brush _tintDefaultBrush = Color.Blue.ToBrush();
+		bool _isDisposed = false;
```

Comment:
Not strictly necessary, but can this be changed from `_isDisposed` to `_disposed` for consistency with our pattern everywhere else?